test/ctia/http/generative/fulltext_search_test.clj

```diff
@@ -140,13 +140,13 @@
                              (utils/update-items
                               incidents
                               ;; update only the first, leave the rest unchanged
-                              #(assoc % :discovery_method "Log Review"
+                              #(assoc % :short_description "Log Review"
```

Comment:
please preserve original version, the changes you made should not impact this.

Reply:
> changes you made should not impact this.

They do though, remember, we've limited Incident searchable_fields, `discovery_method` and `assignees` are no longer in `ctia.entity.incident/searchable-fields`, I had to change it.

Reply:
then it would be nice to replace a token field by another token field

Reply:
Eh? I'm not sure what you mean, would you elaborate?